src/uint256.h

```diff
@@ -48,4 +48,19 @@ inline Data store(const uint256_t& v) {
     return bytes;
 }
 
+// Append a uint256_t value as a big-endian byte array into the provided buffer, and limit
+// the array size by digit/8.
+inline void appendBigEndian(Data &data, const uint256_t &value, uint32_t digit) {
```

Comment:
for consistency, how about `appendBigEndian` -> `encode256BE(value, digit, data)`?

Reply:
It's ok.